### DISCUSSION TITLE
Add an ItemsService option to skip revision-tracking

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#25059] [Feature Requests]
### Describe the Improvement

Similar on how the event-emitting & cache-clearing can be controlled by the ItemsService options-param I'd suggest to also allow bypassing the revision tracking.

This would allow to track revisions made directly by users but skip them on programmatically changes.

For the implementation a new flag would be need in the MutationOptions and then checked in the Service (same as with `emitEvents`):
https://github.com/directus/directus/blob/main/api/src/services/items.ts#L822

**Comment by ComfortablyCoding** [comment]
As this is a quasi Feature Request moving to discussions to further flush out the details as needed.

P.S. The current workaround is to use the underlying `knex` connection to make the necessary questions.

**Comment by ComfortablyCoding** [comment]
This should now be possible via the `skipTracking` option in `ItemsService`

**Reply by Dominic-Marcelino** [reply]
Thanks for letting me know!
